monai/utils/medical_image_converter.py

```diff
@@ -34,6 +35,38 @@ def contain_dicom(path):
     return False
 
 
```

Comment:
Thanks for your update here.
Actually, I think you can treat this converter tool as a temporary reference code and move all the data converter logic into other transforms(or new transforms) after "LoadDICOM".
When all logic is ready in transforms, we can delete this tool and users can use transforms to load DICOM directly.
Thanks.

Reply:
Hi @mhubii ,

Do you want to merge this PR first to my previous DICOM PR first and work on that PR directly?
Thanks.